test/MatrixSets/MatrixZonotope.jl

```diff
@@ -120,9 +121,41 @@ for N in @tN([Float64, Float32, Rational{Int}])
     @test expMZ.M == A_mz
     @test size(expMZ) == size(A_mz)
     @test_throws AssertionError MatrixZonotopeExp(B_mz)
+
+    # vectorize and matrixize
+    c = N[1 0; 0 3]
+    gens = [N[1 -1; 0 2]]
+    MZ = MatrixZonotope(c, gens)
+    Z = vectorize(MZ)
+    @test Z == Zonotope(N[1, 0, 0, 3], hcat(N[1, 0, -1, 2]))
+    MZ2 = matrixize(Z, (2, 2))
+    @test isapprox(center(MZ), center(MZ2))
+    @test isapprox(generators(MZ), generators(MZ2))
+
+    # order
+    @test order(MZ) == 1//4 && order(MZ) == order(Z)
+
+    # remove redundant generators 
+    MZ2 = MatrixZonotope(c, [N[1 4; 0 -2], N[-1 1; 0 -1], N[1 -1; 0 1]])
+    MZred = remove_redundant_generators(MZ2)
+    @test ngens(MZred) == 2
+    @static if isdefined(@__MODULE__, :Polyhedra)
+        @test isequivalent(vectorize(MZ2), vectorize(MZred)) == true
```

Comment:
This is redundant (`X == true` is the same as `X`).
```suggestion
        @test isequivalent(vectorize(MZ2), vectorize(MZred))
```